### PR TITLE
Update custom-cell.md

### PR DESCRIPTION
**Pull request by woohs**
修改console打印错误

### CODE REVIEW

**Comment by simonguo** [comment]
Deploy preview for *rsuite* processing.

Building with commit 45a30fa96afad8432be5d2a6a741cc6fdea13ebc

https://app.netlify.com/sites/rsuite/deploys/5cd59fa0d0dc170008420206

File: src/components/table/custom-cell.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ class CustomColumnTable extends React.Component {
       checkedKeys: nextCheckedKeys
     });
 
-    console.log(this.table);
+    console.log(this.value);
```
Comment (simonguo) [inline src/components/table/custom-cell.md]: this.value 也是没有值的，你要么就把 console.log 给删除掉

Comment (simonguo) [reply src/components/table/custom-cell.md]: https://github.com/rsuite/rsuite.github.io/blob/master/src/components/table/en/custom-cell.md  英文版也有找个问题

Comment (woohs) [reply src/components/table/custom-cell.md]: 抱歉，我的失误，我将重新提交

**Pull request closed without merging**